src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -25,17 +26,39 @@
     description = "Все методы для работы с пользователями",
 )
 
-class AuthController(private val userService: UserService) {
+class AuthController(
+    private val userService: UserService,
+    val ftService: FTService
+) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line